oe_theme.theme

```diff
@@ -156,6 +156,7 @@ function oe_theme_preprocess_paragraph__oe_contextual_navigation(array &$variabl
  * Implements hook_preprocess_menu_local_tasks().
  */
 function oe_theme_preprocess_menu_local_tasks(array &$variables): void {
+  uasort ($variables['primary'], '_oe_theme_weight_comparator');
```

Comment:
We should use `SortArray::sortByWeightElement` instead, check docs [here](https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Component%21Utility%21SortArray.php/function/SortArray%3A%3AsortByWeightElement/8.2.x).

Reply:
@ademarco thank you for the hint, due to the `#weight` property we finally needed [`SortArray::sortByWeightProperty`](https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Component%21Utility%21SortArray.php/function/SortArray%3A%3AsortByWeightProperty/8.2.x)

The changes is made.